app/models/aggregated_calls_received_metric.rb

```diff
@@ -17,6 +17,10 @@ def initialize(organisation, time_period)
       end
   end
 
+  def is_applicable?
```

Comment:
This is a minor style point, but I'd suggest that Ruby's style is to not have an `is_` prefix and instead use the `?` suffix to indicate a boolean method. However readability is the overriding concern, so if you feel like it reads better then we can leave it.